tests/run/cpp_class_attrib.srctree

```diff
@@ -0,0 +1,159 @@
+# mode: run
+# tag: cpp
+
+"""
+PYTHON setup.py build_ext --inplace
+PYTHON -c "import runner"
+"""
+
+######## setup.py ########
+
+from Cython.Build.Dependencies import cythonize
+from distutils.core import setup
+import os
+
+ext_modules= cythonize("rect.pyx")
+setup(ext_modules=ext_modules)
+
+# Copy content from docs/examples/userguide/wrapping_CPlusPlus
```

Comment:
It would be cool if we didn't have to copy the content, but could use the actual files here. They are already extracted into `docs/examples/`

Reply:
I tried to `include` them in the [previous commit](https://github.com/cython/cython/pull/2970/commits/ab792a379c5379af30f874e505db3f481171ee05#diff-0621e9175097afedc8c9279648929396R10) but the relative path didn't work. Is there a pattern to follow to get the base directory in tests?

Reply:
`srctree` tests are extracted into a temporary directory, usually outside of the project directory.

I wouldn't mind passing them the project root directory as an environment variable from the test runner, though, e.g. `CYTHON_PROJECT_DIR`.